### ISSUE TITLE
Cannot find module './BrowserStackReporter'

### ISSUE THREAD

**Issue by bengummer** [karma-runner/karma-browserstack-launcher#75]
[v1.1.0](https://github.com/karma-runner/karma-browserstack-launcher/releases/tag/v1.1.0) has an incorrect path in it which prevents any Browserstack tests from running.

```
ERROR [plugin]: Error during loading "myrepo/node_modules/karma-browserstack-launcher" plugin:
  Cannot find module './BrowserStackReporter'
```

The line responsible for this is `index.js`, specifically

```
var BrowserStackReporter = require('./BrowserStackReporter')
```

which should be

```
var BrowserStackReporter = require('./browserstack-reporter')
```

**Comment by bengummer** [comment]
cc @dignifiedquire this is a bug fix from the 1.1.0 release

**Comment by dignifiedquire** [comment]
Released in 1.1.1
